src/main/java/org/opensearch/security/filter/SecurityRestFilter.java

```diff
@@ -138,6 +145,42 @@ private boolean userIsSuperAdmin(User user, AdminDNs adminDNs) {
         return user != null && adminDNs.isAdmin(user);
     }
 
+    private boolean authorizeRequest(RestHandler original, RestRequest request, RestChannel channel, User user) throws Exception {
+        if (original instanceof RestSendToExtensionAction) {
```

Comment:
Lets remove this check, we aren't using the `RestSendToExtensionAction` for anything, no reason any other named route couldn't be used if it was added by a plugin